x-pack/winlogbeat/module/security/test/testdata/security-windows2016_4757.evtx.golden.json

```diff
@@ -32,7 +32,10 @@
     "user": {
       "domain": "WLBEAT",
       "id": "S-1-5-21-101361758-2486510592-3018839910-500",
-      "name": "Administrator"
+      "name": "Administrator",
+      "target": {
+        "name": "Administrator"
```

Comment:
Seems like a `target.group` could be useful in this case, like`TargetUserName -> target.user.group`.